bika/lims/content/sampletype.py

```diff
@@ -198,6 +254,158 @@ def ContainerTypesVocabulary(self):
         from bika.lims.content.containertype import ContainerTypes
         return ContainerTypes(self, allow_blank=True)
 
+    def getDefaultSmallSticker(self):
+        """
+        Returns the small sticker ID defined as default.
+
+        :return: A string as an sticker ID
+        """
+        values = self.getField('AdmittedStickerTemplates').get(self)
+        if not values:
+            return ''
+        value = values[0].get('small_default')
+        return value
+
+    def getDefaultLargeSticker(self):
+        """
+        Returns the large sticker ID defined as default.
+
+        :return: A string as an sticker ID
+        """
+        values = self.getField('AdmittedStickerTemplates').get(self)
+        if not values:
+            return ''
+        value = values[0].get('large_default')
+        return value
```

Comment:
Almost the same exact logic as `getDefaultSmallSticker`. Better to create a function like
```python
def getDefaultSticker(self, size=STICKER_SMALL):
    ...
    return values[0].get(size, '')
```